src/scss/components/optional/_cuisines-collage.scss

```diff
@@ -0,0 +1,133 @@
+/**
+* Components > Cuisines Collage
+* =================================
+*
+* Example: Homepage – Popular cuisines collage.
+*
+* The `c-cuisinesCollage` component is an optional mixin within Fozzie.
+* If you'd like to use it in your project you can include it by adding `@include cuisinesCollage();` into your SCSS dependencies file.
+*
+* Documentation:
+* https://fozzie.just-eat.com/styleguide/ui-components/cuisines-collage
+*/
+
+
+@mixin cuisinesCollage() {
+    @include media-context(('narrow-mid': 570px, 'mid-wide': 960px, 'wide-huge': 1240px)) {
+        .c-cuisinesCollage-items {
+            width: 100%;
+            height: 330px;
+            display: flex;
+            flex-flow: row nowrap;
+            margin-bottom: spacing(x2);
+            justify-content: center;
+            align-items: flex-start;
+            align-content: normal;
+
+            button {
+                display: block;
+                width: 100%;
+            }
+
+            img {
+                border-radius: $border-radius;
+                object-fit: cover;
+            }
+        }
+
+        .c-cuisinesCollage-btn {
+            max-width: 392px;
+            width: 100%;
+        }
+
+        .c-cuisinesCollage-item {
+            margin-right: spacing(base);
+        }
+
+        .c-cuisinesCollage-item:first-child {
+            img {
+                max-height: 329px;
+                max-width: 272px;
+            }
+        }
+
+        .c-cuisinesCollage-item:nth-child(2) {
+            img {
+                height: 171px;
+            }
+
+            button:first-child img {
+                height: 150px;
+                margin-bottom: 6px;
+            }
+        }
+
+        .c-cuisinesCollage-item:nth-child(3) {
+            img {
+                max-height: 329px;
+                max-width: 254px;
+            }
+        }
+
+        .c-cuisinesCollage-item:last-child {
+            margin-right: 0;
+
+            img {
+                height: 161px;
+                width: 100%;
+                max-width: 461px;
+            }
+
+            button:first-child img {
+                margin-bottom: 5px;
+            }
+        }
+
+        @include media('<wide-huge') {
```

Comment:
Is it better to have this nested within line 65? Keeps an element and all responsive styles in one place.

Reply:
There's a few places throughout that may be better to re-shuffle and have the responsive styles live closer to the elements they are changing 🤔